crypto/keys/lazy_keybase.go

```diff
@@ -14,17 +14,18 @@ var _ Keybase = lazyKeybase{}
 
 // NOTE: lazyKeybase will be deprecated in favor of lazyKeybaseKeyring.
 type lazyKeybase struct {
-	name string
-	dir  string
+	name    string
+	dir     string
+	options []KeybaseOption
 }
 
 // New creates a new instance of a lazy keybase.
-func New(name, dir string) Keybase {
+func New(name, dir string, opts ...KeybaseOption) Keybase {
```

Comment:
Please add a testcase in `lazy_keybase_test.go` to test this API change and ensure that the new `KeybaseOption`s are passed down as expected.

Reply:
Test was a little finicky to setup because of having to override the codec, let me know if this is sufficient enough (covers the whole flow necessary, but could want something specific)

Reply:
Looks good, resolving this.